src/main/java/uk/co/ramp/covid/simulation/population/Population.java

```diff
@@ -377,6 +377,12 @@ public void timeStep(Time t, DailyStats dStats, ContactsWriter contactsWriter) {
                 t = t.advance();
             }
             households.forEach(Household::dayEnd);
+
+            // At the end of each day we also determine possible hospital visits for the next day
```

Comment:
Does this mean there will never be any hospital visits on the first day simulated?